src/aind_metadata_service/sharepoint/client.py

```diff
@@ -98,6 +100,12 @@ class WellType(Enum):
         WHC_NP = "WHC NP"
         OTHER = "Other"
 
+    class SurgeryOrder(Enum):
+        """Enum class for Initial/FollowUp classifiers"""
+
+        INITIAL = "Initial Surgery"
+        FOLLOW_UP = "Follow up Surgery"
```

Comment:
can we just call this second surgery? Follow up to me really implies like something went wrong with teh first surgery, while this really is usually just a separate, second surgery.

Reply:
sure, i'll go ahead and do that